server/app.ts

```diff
@@ -0,0 +1,56 @@
+declare var require: any
+const fs = require('fs')
+const path = require('path')
+const express = require('express')
+const compression = require('compression')
+const bodyParser = require('body-parser')
+
+import { graphqlExpress, graphiqlExpress } from 'graphql-server-express'
+import { makeExecutableSchema } from 'graphql-tools'
+
+import { PORT } from './common'
+import * as plugins from './plugins/api'
+import { GenericNotifier } from './plugins/api/GenericNotifier'
+import { APIReturn } from './plugins/api/APIReturn'
+
+const typeDefs = fs.readFileSync(path.resolve(__dirname, "../api.graphql"), "utf8");
+
+const app = express();
+
+app.use(compression());
+process.on("unhandledRejection", err => {
+  throw err;
+});
+
+type IStatusReturn = { [medium: string]: APIReturn }
+
+const resolvers = {
+  Query: {
+    send_message: (prev: any, args: any) => {
+      let statusRet: IStatusReturn = {}
```

Comment:
be consistent on use of semicolons, maybe add this to tslint. @petschekr do we do this?

Reply:
ALWAYS USE SEMICOLONS. Also please use double quotes instead of single quotes.